nipype/interfaces/base.py

```diff
@@ -634,16 +635,16 @@ def __deepcopy__(self, memo):
             return memo[id_self]
         dup_dict = deepcopy(self.get(), memo)
         # access all keys
-        for key in self.copyable_trait_names():
-            if key in self.__dict__.keys():
-                _ = getattr(self, key)
+        # for key in self.copyable_trait_names():
+        #     if key in self.__dict__.keys():
+        #         _ = getattr(self, key)
         # clone once
         dup = self.clone_traits(memo=memo)
-        for key in self.copyable_trait_names():
-            try:
-                _ = getattr(dup, key)
-            except:
-                pass
+        # for key in self.copyable_trait_names():
```

Comment:
dynamic traits won't work without these. i'm not sure we have a test for it but this is the relevant issue and PR.

https://github.com/enthought/traits/pull/373